main.go

```diff
@@ -119,7 +123,7 @@ func createOrUpdateTargetsAndJobs(target string, t *shield.Target, c *shield.Cli
 	j := &shield.Job{
 		Name:       "Daily",
 		Schedule:   "daily 4am",
```

Comment:
Let's go ahead and parameterize Schedule (`cf protect foo --at 'schedule'`) and Name (`cf protect foo --name `Daily`), with appropriate defaults.

It would also be nice to make paused / unpaused a flag, `--unpaused` and default to pausing the job, like we do right now.  If `--paused` is a more explicit thing to do the default, and take precedence, so be it.